HIP/hip-1082.md

```diff
@@ -0,0 +1,248 @@
+---
+hip: 1082
+title: Allow Setting of EVM Address by EOA
+author: Nana Essilfie-Conduah <@Nana-EC>
+working-group: Richard Bair <@rbair>, Atul Mahamuni <@atul-hedera>, Ty Smith <ty-swirldslabs>, Serg Metelin <@sergmetelin>, Ali Nikan <@alinik4n>, Greg Scullard <@gregscullard>
+requested-by: Hedera community
+type: Standards Track
+category: Service
+needs-council-approval: Yes
+status: Draft
+created: 11-18-24
+discussions-to: https://github.com/hashgraph/hedera-improvement-proposal/discussions/1070
+updated: 01-21-25
+replaces: 631
+requires: 583
+---
+
+## Abstract
+
+Permits accounts without an alias to set one.
+
+[HIP 583](https://hips.hedera.com/hip/hip-583) opened the doors for greater account compatibility with EVM flows by
+utilizing the account alias to encompass the EVM address. However, given the static nature of the alias and concerns of
+complexity, existing accounts with a missing alias value were not permitted to set their alias.
+This HIP aims to rectify this by allowing EOAs who have no alias set to set it to an ECDSA key derived evm address
+value they can prove ownership of.
+
+This HIP also replaces the need for [HIP 631: Account Virtual Addresses](https://hips.hedera.com/hip/hip-631) as it
+aimed to resolve this issue as well as provide even greater functionality. A form of HIP 631 could be revisited in the
+future but not with the goal of providing address equivalence with the EVM.
+
+## Motivation
+
+On Ethereum, every account is addressed by an address which is derived from an ECDSA public key. The
+EVM also has a specific precompile function, `ecrecover`, that given an ECDSA signature can extract the public key
+and convert that public key into an address. This allows an EVM program to figure out the address of
+transaction signers.
+Hiero accounts are not limited to ECDSA keys, and Hiero accounts can rotate their keys. Both of these
```

Comment:
This section and the following two need some additional blank lines to introduce paragraphing in the rendered version.